### ISSUE TITLE
Generic Embed Type/Other Web Video Types?

### ISSUE THREAD

**Issue by vincent404** [cosmicradiotv/podcast-plugin#13]
So I noticed that we have a Youtube type in the database. I'm wondering if we should add other popular web video hosting (vimeo, dailymotion, etc) and maybe a generic embed type? Might be overkill. Thoughts?

**Comment by blendermf** [comment]
I agree with adding other hosts, and maybe an embed (slightly harder because releases currently take urls, but that could probably be changed to hold a url/embed code depending on the release type, we could even add a radio control for release types to say this takes either a url or embed code). I think this might be something we worry about a little down the road (especially since it's pretty easy to add another video type at least in the code, that's something we should document. The youtube player we implement could basically be the example of how to create that)
